doc/Doxyfile.in

```diff
@@ -102,6 +110,9 @@ EXCLUDE_SYMLINKS       = NO
 EXCLUDE_PATTERNS       = *.inl.hpp *.impl.hpp *_detail.hpp */cudev/**/detail/*.hpp
 EXCLUDE_SYMBOLS        = cv::DataType<*> int void
 EXAMPLE_PATH           = @CMAKE_DOXYGEN_EXAMPLE_PATH@
+EXAMPLE_PATH          += @CMAKE_CURRENT_SOURCE_DIR@/aliases/add_toggle.html
```

Comment:
Can we add this scripts to some JS file, or at least to the page header? As I understand, currently this snippet will be inserted each time we add new button.